robotoff/app/core.py

```diff
@@ -134,6 +134,9 @@ def get_insights(
 
         elif order_by == "n_votes":
             query = query.order_by(ProductInsight.n_votes.desc())
+    else:
+        query = query.order_by(ProductInsight.id.desc())
```

Comment:
As said in meeting: better not introduce a default sorting in a widely used function, it might have unexpected side effects.

We should instead use `order_by="most_recent"` in Collection view and implement it here.

Also you should use timestamp here and not id, because id is special on ProductInsight (it's not numerical, it's a uuid).